genie-core/src/main/java/com/netflix/genie/core/util/MetricsUtils.java

```diff
@@ -94,21 +86,25 @@ private MetricsUtils() {
     /**
      * Convenience method to add common tags from job execution environment, for
      * slicing and dicing metrics, to the map of tags passed in.
+     * Populates NO_COMMAND_FOUND if command is missing
+     * Populates NO_CLUSTER_FOUND if cluster is missing
      * Populates NO_ID_FOUND in case ids are not present in the job execution environment
      * E.g NO_ID_FOUND will be added to the tags maps for the cluster_id key in case cluster_id
      * does not exist in jobExecEnv
      *
      * @param tags       The map of tags
      * @param jobExecEnv Genie job execution env
      */
-    public static void addCommonJobWorkflowMetricTags(final JobExecutionEnvironment jobExecEnv,
-                                                      final Map<String, String> tags) {
-        addCommonJobWorkflowMetricTags(jobExecEnv.getCluster(), jobExecEnv.getCommand(), tags);
+    public static void addClusterAndCommandTags(final JobExecutionEnvironment jobExecEnv,
+                                                final Map<String, String> tags) {
```

Comment:
Following code convention for the rest of this codebase, the `) {` goes on a newline